services/route/server.go

```diff
@@ -13,25 +13,27 @@ import (
 	"github.com/signadot/hotrod/pkg/delay"
 	"github.com/signadot/hotrod/pkg/notifications"
 	"github.com/signadot/hotrod/pkg/tracing"
+	"github.com/signadot/hotrod/services/route/github.com/signadot/hotrod/services/route"
```

Comment:
@jmsktm, why do we need this, instead of using the local module?

Reply:
This was a result of how I was generating the source earlier. I have now submitted a commit to this pull request containing code generated with the `make` command.

Reply:
Can you check if this addresses the issue you were facing with `go mod tidy`?

Reply:
Yes, works fine now, thanks